nsist/pypi.py

```diff
@@ -250,12 +252,58 @@ def extract_wheel(whl_file, target_dir, exclude=None):
     shutil.rmtree(str(td))
 
 
-def fetch_pypi_wheels(requirements, target_dir, py_version, bitness,
-                      extra_sources=None, exclude=None):
-    for req in requirements:
+def fetch_pypi_wheels(wheels_requirements, wheels_paths, target_dir, py_version,
+                      bitness, extra_sources=None, exclude=None):
+    """
+    Gather wheels included explicitly by wheels_pypi parameter 
+    or matching glob paths given in local_wheels parameter.
+    """
+    distributions = []
+    # We try to get the wheels from wheels_pypi requirements parameter
+    for req in wheels_requirements:
         wl = WheelLocator(req, py_version, bitness, extra_sources)
-        whl_file = wl.fetch()
+        whl_file = wl.fetch() 
         extract_wheel(whl_file, target_dir, exclude=exclude)
+        distributions.append(wl.name)
+    # Then from the local_wheels paths parameter
+    for glob_path in wheels_paths:
+        paths = glob.glob(glob_path)
+        if not paths:
+            raise ValueError('Error, glob path {0} does not match any wheel file'.format(glob_path))
+        for path in glob.glob(glob_path):
```

Comment:
This could use `for path in paths` to save doing the globbing again. It won't be a big deal in most cases, but a complex glob pattern could involve scanning a lot of directories.

Reply:
pip install requests requests_download jinja2 yarg nose win_cli_launchers testpath

Reply:
And corrected.